hawkular-inventory-api/src/test/java/org/hawkular/inventory/api/test/AbstractBaseInventoryTestsuite.java

```diff
@@ -2554,21 +2569,21 @@ public void testObserveIdentityHashChangedOnParentOfDeletedEntity() throws Excep
         });
     }
 
-    private void testIdentityHashObservation(String tenantId, Consumer<Map<Class<?>, Integer>> test) throws Exception {
+    private void testSyncHashObservation(String tenantId, Consumer<Map<Class<?>, Integer>> test) throws Exception {
         Subscription subs = null;
         try {
-            createInventoryForIdentityHashChangeObservations(tenantId);
+            createInventoryForSyncHashChangeObservations(tenantId);
 
             Map<Class<?>, Integer> updateCounts = new HashMap<>();
 
             subs = Observable.merge(
-                    inventory.observable(Interest.in(Feed.class).having(identityHashChanged())),
-                    inventory.observable(Interest.in(ResourceType.class).having(identityHashChanged())),
-                    inventory.observable(Interest.in(MetricType.class).having(identityHashChanged())),
-                    inventory.observable(Interest.in(OperationType.class).having(identityHashChanged())),
-                    inventory.observable(Interest.in(Resource.class).having(identityHashChanged())),
-                    inventory.observable(Interest.in(Metric.class).having(identityHashChanged())),
-                    inventory.observable(Interest.in(DataEntity.class).having(identityHashChanged()))
+                    inventory.observable(Interest.in(Feed.class).having(syncHashChanged())),
+                    inventory.observable(Interest.in(ResourceType.class).having(syncHashChanged())),
+                    inventory.observable(Interest.in(MetricType.class).having(syncHashChanged())),
+                    inventory.observable(Interest.in(OperationType.class).having(syncHashChanged())),
+                    inventory.observable(Interest.in(Resource.class).having(syncHashChanged())),
+                    inventory.observable(Interest.in(Metric.class).having(syncHashChanged())),
+                    inventory.observable(Interest.in(DataEntity.class).having(syncHashChanged()))
```

Comment:
actually, who can be interested in those hash change events? Is it intentended that agent listen on this and calls the sync if something has changed?